scss/components/_badge.scss

```diff
@@ -3,6 +3,19 @@
 @use '../variables/transition';
 
 @use '../functions' as fn;
+@use '../mixins' as mx;
+
+/// Colors for different badge types
+/// Sorted by background, text and border color
+/// @type Map
+$badge-types: (
+	primary: (color.$primary, color.$primary-50, color.$primary-600),
```

Comment:
Würde hier dasselbe wie in diesem PR fragen:
https://github.com/hyperlinkgroup/blower/pull/4/files/196b1bd1f9ca93ce4817e947be05d72fd25867ea#diff-4f565b0cfef0c4581f187d2a66dd017975f7b3f8c12e75486058a9169c80deb3